### PR TITLE
feat(ios): bump minimum deployment target to iOS11

### PR DESCRIPTION
**Pull request by wwwcg**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

**Review by churchill-zhang — APPROVED** [review]

File: hippy.podspec [hunk 1]
```diff
@@ -29,15 +29,17 @@ Pod::Spec.new do |s|
     ss.libraries = 'c++'
     ss.framework = 'JavaScriptCore'
     ss.dependency 'hippy/coreThirdParty'
-    ss.source_files = ['core/include/**/*.{h,cc}', 
+    ss.source_files = ['core/include/**/*.{h,m,c,mm,s,cpp,cc}',
```
Comment (ozonelmy) [inline hippy.podspec:32]: core里面有m,c,mm,s,cpp这些文件吗

File: ios/sdk/module/turbo/HippyOCTurboModule+Inner.h [hunk 2]
```diff
@@ -25,13 +25,36 @@
 #ifdef __cplusplus
 
 #import "HippyOCTurboModule.h"
+#include <memory>
+#include "core/napi/jsc/jsc_ctx.h"
+#include "core/napi/jsc/jsc_ctx_value.h"
```
Comment (ozonelmy) [inline ios/sdk/module/turbo/HippyOCTurboModule+Inner.h:30]: 头文件里最好用前置声明，尽量不直接包含头文件

**Review by ilikethese — APPROVED** [review]

File: hippy.podspec (hunk 1 above)
Comment (wwwcg) [reply hippy.podspec:32]: 本来不想修改的，但新加的文件里有.cpp后缀，为了防止后续再被过滤，统一加上了

File: ios/sdk/module/turbo/HippyOCTurboModule+Inner.h (hunk 2 above)
Comment (wwwcg) [reply ios/sdk/module/turbo/HippyOCTurboModule+Inner.h:30]: 直接移动的polly的代码，这部分主要是为了兼容OC的modular import。 @churchill-zhang

**Review by churchill-zhang — APPROVED** [review]

**Review by ilikethese — APPROVED** [review]

**Review by ozonelmy — APPROVED** [review]
LGTM

**Pull request merged**